src/contract.rs

```diff
@@ -71,4 +71,15 @@ impl DIDTrait for DIDContract {
     fn get_did(e: Env) -> DIDDocument {
         storage::read_did_document(&e)
     }
+
+    fn upgrade(e: Env, new_wasm_hash: BytesN<32>) {
+        let admin = storage::read_admin(&e);
+        admin.require_auth();
+
+        e.deployer().update_current_contract_wasm(new_wasm_hash);
+    }
+
+    fn version(e: Env) -> String {
+        String::from_str(&e, "0.5.0")
+    }
```

Comment:
What do you think about setting the version as a module const and using it here?

Reply:
@MarioRodriguezS I find a way to return the version specified at Cargo.toml file. I think it's better that way! WDYT?